src/View/BakeView.php

```diff
@@ -142,7 +138,10 @@ protected function _paths(?string $plugin = null, bool $cached = true): array
     {
         $paths = parent::_paths($plugin, false);
         foreach ($paths as &$path) {
-            $path .= static::BAKE_TEMPLATE_FOLDER . DS;
+            // Append 'bake' to all directories that aren't the application override directory.
```

Comment:
So bake themes would still use path `PLUGIN_ROOT/templates/bake/*` right?

Reply:
Yea. It seems we didn't have a test-case for overriding bake template from the application templates. Only theme plugins.

Reply:
Yes bake themes and plugin tasks are unchanged by this. This change is only relevant to application overrides.